### ISSUE TITLE
projectcontour.io has a poor accessibility score

### ISSUE THREAD

**Issue by SDBrett** [projectcontour/contour#1819]
An audit using Google Lighthouse has returns a poor score for the sites accessibility. Many of the failing items are quite basic, such as poor contrast ratios.

I've exported the report and uploaded the HTML to a gist https://gist.github.com/SDBrett/950a957e51c77025fcb475aea6d463fd#file-projectcontour-io-20191029t121902-html

**Comment by SDBrett** [comment]
Would it be worthwhile investing time into integrating lighthouse into the PR process to automatically validate the site for items such as accessibility?
